src/test/kotlin/com/aa/airportagent/flight/FlightInterfaceTest.kt

```diff
@@ -0,0 +1,54 @@
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+class FlightInterfaceTest : FreeSpec({
+    val flightInterface = FlightInterfaceImp()
+    "totalFlightsByStation" {
+        var flights = listOf(
+            Flight(1, LocalDate.of(2022, 1, 1), 1, "PHX", "DFW", "001", Status.SCHEDULED,
+                OffsetDateTime.of(2022, 1, 1, 12, 0, 0, 0, ZoneOffset.UTC),
+                OffsetDateTime.of(2022, 1, 1, 15, 0, 0, 0, ZoneOffset.UTC)
+            ),
+            Flight(2, LocalDate.of(2022, 1, 1), 2, "DFW", "CLT", "001", Status.SCHEDULED,
+                OffsetDateTime.of(2022, 1, 1, 12, 0, 0, 0, ZoneOffset.UTC),
+                OffsetDateTime.of(2022, 1, 1, 15, 0, 0, 0, ZoneOffset.UTC)
+            )
+        )
+        flightInterface.totalFlightsByStation(flights) shouldBe mapOf("PHX" to 1, "DFW" to 2, "CLT" to 1)
+    }
+    "firstFlightOfDay" - {
```

Comment:
Move all this test data into another class or some place it doesn't distract from the testing code